### ISSUE TITLE
Deprecated method 'exists?' for File:Class in ruby 3.2.0

### ISSUE THREAD

**Issue by OleStan** [rjocoleman/capistrano-upload-config#14]
On ruby 3.2.0 method `exists?` is removed, and now command such `cap production config:init` return error
```
cap aborted!
NoMethodError: undefined method `exists?' for File:Class
```

Method `File.exists?` used in lib/capistrano/tasks/upload-config.rake

---

### Steps to reproduce

1. Set project ruby version 3.2.0
2. Setup Capistrano
3. try to use command `cap production config:init` 

### System configuration
Ruby 3.2.0
Rails 7.0.4
Capistrano 3.17.1
Capistrano-Upload-Config 0.8.2

**Comment by rjocoleman** [comment]
It's changed to `exist?` in Ruby 3.2.0 (`exists?` has been deprecated). 

If you want to send a PR i can merge and cut a new release, else I might be able to get to it next week - thanks for the report!

**Comment by rjocoleman** [comment]
closed via #15

**Comment by rjocoleman** [comment]
New release is live thanks @OleStan https://rubygems.org/gems/capistrano-upload-config/versions/0.9.0
